storm-client/src/jvm/org/apache/storm/daemon/worker/WorkerState.java

```diff
@@ -717,11 +718,20 @@ private Assignment getLocalAssignment(IStormClusterState stormClusterState, Stri
 
         IWaitStrategy backPressureWaitStrategy = IWaitStrategy.createBackPressureWaitStrategy(topologyConf);
         Map<List<Long>, JCQueue> receiveQueueMap = new HashMap<>();
+
+        Map<Integer, String> taskIdToCompId = StormCommon.stormTaskInfo(topology, topologyConf);
```

Comment:
Is it possible to reuse `this.taskToComponent`?

Reply:
Yes, updated. Thanks for pointing out.